templates/dhcpd.conf.j2

```diff
@@ -10,9 +10,6 @@ omapi-port {{ dhcp_omapi_port }};
 
 option domain-name "{{ dhcp_common_domain }}";
 {% endif %}
-{% if dhcp_common_domain_search is defined %}
```

Comment:
Can you check why this was changed?

Reply:
Many of the parameters have been removed to match with existing dhcp.conf file of epouta-gw1. You can compare by logging both of the servers and comparing /etc/dhcp/dhcpd.conf file.